tests/xharness/Jenkins/Jenkins.cs

```diff
@@ -15,6 +15,36 @@
 using Xharness.Jenkins.Reports;
 
 namespace Xharness.Jenkins {
+	[Flags]
+	public enum TestSelection {
```

Comment:
I have a small itch with your enum, we are limited to 30~ values before we overflow and we are currently using 24 slots if I counted correctly, I guess this is minor and since we are not shipping this it can be changed anytime.